pkg/util/funcutil/placeholdergroup.go

```diff
@@ -83,14 +82,10 @@ func fieldDataToPlaceholderValue(fieldData *schemapb.FieldData) (*commonpb.Place
 			return nil, errors.New("vector data is not schemapb.VectorField_SparseFloatVector")
 		}
 		vec := vectors.SparseFloatVector
-		bytes, err := proto.Marshal(vec)
```

Comment:
What is the reason for removing this `Marshal`?

Reply:
it was a mistake: `PlaceholderValue::Values` should be just the byte representation of sparse vector, not some serialized proto. no one has ever used this search by pk feature thus this bug is never triggered.

Reply:
it was a mistake: PlaceholderValue::Values should be just the byte representation of sparse vector, not some serialized proto. no one has ever used this search by pk feature thus this bug is never triggered.